controller_manager/test/test_controllers_chaining_with_controller_manager.cpp

```diff
@@ -390,6 +454,14 @@ class TestControllerChainingWithControllerManager
   double EXP_RIGHT_WHEEL_HW_STATE = 0.0;
   double EXP_LEFT_WHEEL_REF = 0.0;
   double EXP_RIGHT_WHEEL_REF = 0.0;
+
+  // Expected behaviors struct used in chaining activation/deactivation tests
+  struct expectedBehaviorStruct
```

Comment:
We are usually using capitalized begin of the struct names. Can you please update that?

Reply:
Done!